ruby-sdk/lib/eppo_client/config.rb

```diff
@@ -8,7 +8,15 @@ module EppoClient
   class Config
     attr_reader :api_key, :assignment_logger, :base_url, :poll_interval_seconds, :poll_jitter_seconds, :log_level
 
-    def initialize(api_key, assignment_logger: AssignmentLogger.new, base_url: EppoClient::Core::DEFAULT_BASE_URL, poll_interval_seconds: EppoClient::Core::DEFAULT_POLL_INTERVAL_SECONDS, poll_jitter_seconds: EppoClient::Core::DEFAULT_POLL_JITTER_SECONDS, initial_configuration: nil, log_level: nil)
+    def initialize(
+      api_key,
+      assignment_logger: AssignmentLogger.new,
+      base_url: EppoClient::Core::DEFAULT_BASE_URL,
+      poll_interval_seconds: EppoClient::Core::DEFAULT_POLL_INTERVAL_SECONDS,
+      poll_jitter_seconds: EppoClient::Core::DEFAULT_POLL_JITTER_SECONDS,
+      initial_configuration: nil,
+      log_level: nil
+    )
```

Comment:
just broke args in separate lines here. Still TBD on `initial_configuration` being unused in `initialize`